packages/shared/client/ReplayClient.ts

```diff
@@ -159,6 +162,10 @@ export class ReplayClient implements ReplayClientInterface {
     return response;
   }
 
+  async ensureSourcesLoaded(): Promise<void> {
+    await this._threadFront.ensureAllSources();
+  }
```

Comment:
Can we name this something _more explicit_ like `waitForLoadingSources`?

To me, the name "ensure loaded" implies it will:
1.  Check if the thing is already loaded
2. If not, check if it's still loading (and wait)
3. If not, start loading it (and wait)

It doesn't do that 3rd thing though.